fluss-client/src/test/java/org/apache/fluss/client/table/scanner/log/DefaultCompletedFetchTest.java

```diff
@@ -321,6 +356,18 @@ void testComplexTypeFetch() throws Exception {
                     .isEqualTo(((Object[]) ((Object[]) complexData.get(i)[3])[1])[1]);
             assertThat(nestedRow.getString(2).toString())
                     .isEqualTo(((Object[]) complexData.get(i)[3])[2]);
+
+            assertThat(row.getMap(4)).isInstanceOf(GenericMap.class);
+            GenericMap simpleMap = (GenericMap) row.getMap(4);
+            assertThat(simpleMap.size()).isGreaterThan(0);
+
+            assertThat(row.getMap(5)).isInstanceOf(GenericMap.class);
+            GenericMap nestedMap = (GenericMap) row.getMap(5);
+            assertThat(nestedMap.size()).isGreaterThan(0);
+
+            assertThat(row.getMap(6)).isInstanceOf(GenericMap.class);
+            GenericMap mapWithArray = (GenericMap) row.getMap(6);
+            assertThat(mapWithArray.size()).isGreaterThan(0);
```

Comment:
Actually, this approach doesn’t verify the actual field values, and it significantly inflates the line count, making the code harder to maintain.  

Moreover, we’ll soon need to add assertions for column projection here as well.

My suggestion: prepare a list of **expected rows** upfront and perform a direct assert equals comparison with the actual result. 

This keeps the test concise, expressive, and much easier to maintain.